presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerificationManager.java

```diff
@@ -466,32 +461,31 @@ protected Boolean visitTable(Table node, Integer level)
         }
 
         @Override
-        protected Boolean visitTableSubquery(TableSubquery node, Integer level)
+        protected Boolean visitCreateTable(CreateTable node, Integer level)
         {
-            //Incrementing level when we reach a sub-query
-            return process(node.getQuery(), level + 1);
+            return false;
         }
 
         @Override
-        protected Boolean visitCreateTable(CreateTable node, Integer level)
+        protected Boolean visitValues(Values node, Integer level)
         {
             return false;
         }
 
         @Override
-        protected Boolean visitCreateTableAsSelect(CreateTableAsSelect node, Integer level)
+        protected Boolean visitLiteral(Literal node, Integer context)
         {
-            return process(node.getQuery(), level);
+            return false;
         }
 
         @Override
-        protected Boolean visitCreateView(CreateView node, Integer level)
+        protected Boolean visitOrderBy(OrderBy node, Integer context)
         {
-            return process(node.getQuery(), level);
+            return false;
         }
```

Comment:
I can see these got taken out. Can you help me understand how this gets handled in the new code.

Reply:
The getChildren() call in `CreateTableAsSelect` and `CreateView` returns immutable list containing `Query`.
Both `visitCreateTableAsSelect` and `visitCreateView`  methods invoke  `visitStatement(node, context);` which in turn invokes `visitNode(node, context);`. visitNode implementation here processes each child so visitQuery gets invoked